merkledag/merkledag.go

```diff
@@ -23,7 +23,13 @@ func init() {
 	ipld.Register(cid.DagCBOR, ipldcbor.DecodeBlock)
 }
 
+// contextKey is a type to use as value for the ProgressTracker contexts.
+type contextKey string
```

Comment:
I would export this type.

Reply:
It has no use outside. If someone ever has a use, it can be exported then.

Reply:
Yeah, your right.  Sorry.